application/views/admin/super/layout_main.php

```diff
@@ -30,7 +30,7 @@
 
 $containerClass = !Yii::app()->user->isGuest ? 'container-fluid' : 'container-fluid ps-0';
 echo '<!-- Full page, started in SurveyCommonAction::renderWrappedTemplate() -->
-<div class="' . $containerClass . '" id="in_survey_common_action">';
```

Comment:
Are we sure this is not needed any more?

Reply:
As per our chat, I brought back the container class and applied CSS in order to avoid the side paddings in this specific case